device-types/Cisco/ESA-C390.yaml

```diff
@@ -0,0 +1,30 @@
+---
+manufacturer: Cisco
+model: ESA C390
+slug: cisco-esa-c390
+part_number: ESA-C390-K9
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: Management
```

Comment:
Typically it would not be named management, it would typicially be named "GigabitEthernet0" if it is a device running an IOS like system.

If it is a IMC type interface, I would recommend using "CIMC" for the name.

Reply:
The Device is running an on a AsyncOS. 
I had the output of the interfaces sent to me:
```
Currently configured interfaces:
1. Data1_Inside (5.5.5.5/24 on Data 1: something.example.com)
2. Data2_Outside (5.5.5.5/24 on Data 2: something.example.com)
3. Mgmt (5.5.5.5/24 on Management: something.example.com)
```
After that I renamed the interfaces but the management interface is called "Management".
Please feel free to let me know if you see it otherwise.